docs/compass/03-automatic-scenario-assignment.md

```diff
@@ -0,0 +1,213 @@
+# Automatic Scenario Assignment
+
```

Comment:
Add Introduction header

Reply:
according to @klaudiagrz such header is not necessary (it should be used only in the README files)